src/github.com/matrix-org/dendrite/federationapi/writers/send.go

```diff
@@ -159,7 +159,7 @@ func (t *txnReq) processEvent(e gomatrixserverlib.Event) error {
 	// TODO: Check that the event is allowed by its auth_events.
 
 	// pass the event to the roomserver
-	if err := t.producer.SendEvents([]gomatrixserverlib.Event{e}); err != nil {
+	if err := t.producer.SendEvents([]gomatrixserverlib.Event{e}, ""); err != nil {
```

Comment:
Albeit the type documentation for `InputRoomEvent` does state this is valid and says what it does, but it's a bit too far divorced from this call-site for it to be found by people. Heck, the only reason why I found it was because it was part of this diff!